README.md

```diff
@@ -41,29 +34,6 @@ In version **6.x** support for `transitionStyle` property has been dropped. More
 
 `yarn add react-native-pager-view`
 
-## New architecture setup (Fabric)
-
-This library supports new architecture! We have two example folders one for each architecture. If you are using this library in your own project there some extra steps needed. 
```

Comment:
I would leave this info and update the instructions on how to run normal example with fabric enabled

Reply:
I removed it so it matches current README on `main` branch, but I think it will be good idea to create a PR adding fabric instructions to `main` branch and just sync them. Same applies for comment about `yarn bootstrap-fabric` because `main` is also missing that script

Reply:
Okay, let's do it in a separate PR